.github/workflows/awie.yml

```
@@ -92,8 +106,8 @@ jobs:
 
   deliver-sources:
     runs-on: [self-hosted, common]
-    needs: [get-version, package]
-    if: ${{ contains(fromJson('["stable"]'), needs.get-version.outputs.stability) && github.event_name != 'workflow_dispatch' }}
+    needs: [get-version, package, check-version]
```

```suggestion
    needs: [package]
```
isn't package the only one required ?